cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/instantsearch-config.js

```diff
@@ -360,14 +370,25 @@ function enableInstantSearch(config) {
                         if (item.colorVariations) {
                             // Display the swatches only if at least one item has some colorVariations
                             displaySwatches = true;
-                            item.colorVariations.forEach(colorVariation => {
+                            item.colorVariations.forEach((colorVariation, index) => {
```

Comment:
You have once again added something that is not used, please clean up the code before requesting a review.

```suggestion
                            item.colorVariations.forEach(colorVariation => {
```